packages/angular/package.json

```diff
@@ -33,37 +34,34 @@
   "license": "Apache-2.0",
   "author": "IBM",
   "peerDependencies": {
-    "@angular/common": "^9.0.0 || ^10.0.0 || ^11.0.0",
-    "@angular/core": "^9.0.0 || ^10.0.0 || ^11.0.0",
-    "@angular/forms": "^9.0.0 || ^10.0.0 || ^11.0.0",
+
     "carbon-components": "^10.0.0",
     "carbon-components-angular": "^4.0.0",
-    "rxjs": "^6.0.0",
-    "zone.js": "^0.8.26 || ^0.9.0 || ^0.10.0"
+    "rxjs": "^6.0.0"
   },
   "devDependencies": {
-    "@angular-builders/custom-webpack": "11.1.1",
-    "@angular-devkit/build-angular": "0.1101.1",
-    "@angular-devkit/build-ng-packagr": "0.1002.1",
-    "@angular-devkit/core": "11.1.1",
-    "@angular/animations": "11.1.1",
-    "@angular/cli": "11.1.1",
-    "@angular/common": "11.1.1",
-    "@angular/compiler": "11.1.1",
-    "@angular/compiler-cli": "11.1.1",
-    "@angular/core": "11.1.1",
-    "@angular/forms": "11.1.1",
-    "@angular/platform-browser": "11.1.1",
-    "@angular/platform-browser-dynamic": "11.1.1",
-    "@angular/router": "11.1.1",
+    "@angular-devkit/build-angular": "12.2.17",
+    "@angular-devkit/core": "12.2.17",
+    "@angular/animations": "12.2.16",
+    "@angular/cli": "12.2.17",
+    "@angular/common": "12.2.16",
+    "@angular/compiler": "12.2.16",
+    "@angular/compiler-cli": "12.2.16",
+    "@angular/core": "12.2.16",
+    "@angular/forms": "12.2.16",
+    "@angular/platform-browser": "12.2.16",
+    "@angular/platform-browser-dynamic": "12.2.16",
+    "@angular/router": "12.2.16",
```

Comment:
Dev deps (all, not just highlighted) should stay at lower versions to ensure backwards compatibility when adding new things. Integration tests cover everything else.
_It's ok to change them temporarily for testing locally._